yapapi/engine.py

```diff
@@ -42,13 +43,13 @@
 from yapapi.script.command import BatchCommand
 from yapapi.storage import gftp
 from yapapi.strategy import MarketStrategy, SCORE_NEUTRAL
-from yapapi.utils import AsyncWrapper
+from yapapi.utils import AsyncWrapper, yagna_version_less_than
 
-
-DEBIT_NOTE_MIN_TIMEOUT: Final[int] = 30  # in seconds
-"Shortest debit note acceptance timeout the requestor will accept."
-
-DEBIT_NOTE_ACCEPTANCE_TIMEOUT_PROP: Final[str] = "golem.com.payment.debit-notes.accept-timeout?"
+DEFAULT_PROPERTY_VALUE_RANGES = {
+    "golem.com.payment.debit-notes.accept-timeout?": (30.0, None),
+    "golem.com.scheme.payu.debit-note-interval-sec?": (20.0, None),
+    "golem.com.scheme.payu.payment-timeout-sec?": (None, 3600.0),
```

Comment:
Maybe I'm missing something, but didn't we decide that we want **lower** bound here, not **upper** bound?

Reply:
Yes, it's the provider that needs an upper bound, thanks for reminding.

Reply:
Fixed